prow/plugins.yaml

```diff
@@ -236,6 +236,10 @@ slack:
     whitelist:
     - k8s-ci-robot
 
+milestone_applier:
+  kubernetes/test-infra:
```

Comment:
Should we also do for contributor playground? It would be nice to check the behaviour when the base branch isn't the default branch too. :grimacing:

Reply:
Can we do that in a followup? There aren't any branches or milestones over there so I'm not sure what mapping to use